### ISSUE TITLE
Promise is rejected without catch somewhere in tests

### ISSUE THREAD

**Issue by willscott** [oasislabs/web3c.js#99]
when testing 1.0.2, tests all pass, but there is an unhandled promise rejection somewhere:

```
 Web3
    ✓ should support transient contracts with separate key state
    ✓ should retrieve contract keys from a previously deployed contract address (55ms)
    ✓ should not retrieve contract keys from a non deployed contract address
(node:8658) UnhandledPromiseRejectionWarning: AssertionError [ERR_ASSERTION]: Missing expected rejection.
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:8658) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:8658) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    ✓ should deploy a confidential counter contract
    ✓ should execute transactions and calls (52ms)
    ✓ should get confidential getPastLogs logs
    ✓ should estimate gas for confidential transactions the same as gas actually used
    ✓ should yield a larger estimate for confidential transactions than non-confidential
```

Worth tracking down and fixing

**Comment by armaniferrante** [comment]
Looks like this is due to https://github.com/oasislabs/web3c.js/blob/master/test/web3c.js#L69. This only happens for me against the mock gateway, and not against devnet, staging, or a local one.

**Comment by willscott** [comment]
should it be `return await ...` in that async function?

**Comment by armaniferrante** [comment]
Nope that doesn't address the issue.

**Comment by armaniferrante** [comment]
@willscott you are correct, we need the await.

**Comment by armaniferrante** [comment]
Fixed by #100.
